reference/providers/kubernetes-v2/index.md

```diff
@@ -150,6 +150,26 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   If instead Spinnaker is deploying ReplicaSets directly without a Deployment,
   this annotation does the job.
 
+## Traffic
+
+* `traffic.spinnaker.io/load-balancers`
+
+  In Spinnaker 1.10, you can specify which load balancers
```

Comment:
Prefer "*As of* Spinnaker 1.10" (unless you're going to remove this in 1.11+ ;)

But ideally we wouldn't even refrence the version. Just say "You can specify which load balancers..."

This is why I want to enable versioned docs.